java/.gitattributes

```diff
@@ -1,2 +1,3 @@
 .gitattributes export-ignore
 .gitignore export-ignore
+* text eol=lf
```

Comment:
does it mean that all files are to be considerered as text files (vs binary)? Would'nt that cause issues when adding binary files (for testing for example?)

Reply:
Maybe it should be `* text=auto eol=lf`?

Reply:
Good catch. I was following the instruction from spotless.. https://github.com/diffplug/spotless/tree/main/plugin-gradle#line-endings-and-encodings-invisible-stuff